src/components/organisms/OrganismImageGallery.vue

```diff
@@ -26,11 +26,15 @@ export default {
     data() {
         return {
             imageSize: '',
+            spotImage: [
+                'https://parkspot.blob.core.windows.net/assets/default.png',
+            ],
         };
     },
     computed: {
         ...mapState('sdp', {
-            spotImage: (state) => state.image,
+            image: (state) => state.image,
```

Comment:
please rename this to images
to highlight that its an array

Reply:
done